app/stac_api/urls.py

```diff
@@ -40,4 +42,5 @@
         AssetDetail.as_view(),
         name='asset-detail'
     ),
+    path(f"{API_BASE}/get_token", obtain_auth_token, name='get_token'),
```

Comment:
name should be kebabcase like other path names; `name='get-tocken'`

Reply:
Done